src/app/app.component.ts

```diff
@@ -28,6 +28,9 @@ export class MyApp {
       // Here you can do any higher level native things you might need.
       statusBar.styleDefault();
       splashScreen.hide();
+      platform.registerBackButtonAction(()=> {
+
```

Comment:
Take out spaces.

So it'll look like this:

`platform.registerBackButtonAction(()=> {},);`